cmake/presets/iOS.json

```diff
@@ -0,0 +1,109 @@
+{
+  "version": 6,
+  "include": ["common.json"],
+  "configurePresets": [
+    {
+      "name": "iOS",
```

Comment:
A [conditional configuration](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#condition) could be added here because you will not be able to build iOS on platforms but macOS.

```
      "condition": {
        "type": "equals",
        "lhs": "${hostSystemName}",
        "rhs": "Darwin"
      }
```

Reply:
Qt will not be available for iOS on other platforms so I decided already when starting writing the workflows to not complicate them unnecessarily.